source/blog/_sidebar.html.erb

```diff
@@ -65,13 +65,12 @@
   <h3 class="list-group-title" id="browse-subject">Browse by Subject</h3>
   <ul class="list-group" aria-labelledby="browse-subject">
     <li class="list-item"><a href="<%= tag_path 'Announcement'%>">Announcements</a></li>
-    <li class="list-item"><a href="<%= tag_path 'Ember CLI'%>">Ember CLI</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Community'%>">Community</a></li>
     <li class="list-item"><a href="<%= tag_path 'Ember Data'%>">Ember Data</a></li>
-    <li class="list-item"><a href="<%= tag_path 'Performance'%>">Performance</a></li>
+    <li class="list-item"><a href="<%= tag_path 'Leadership'%>">Leadership</a></li>
```

Comment:
What do you think about renaming the tag `Leadership` to `Steering Committee` or something similar? I think this aligns even better with the new wordings from the Ember Community Reorganization announcement:

https://www.emberjs.com/blog/2018/06/30/organizing-our-contributors.html